go.mod

```diff
@@ -34,9 +38,14 @@ require (
 	k8s.io/api v0.21.0-rc.0
 	k8s.io/apimachinery v0.21.0-rc.0
 	k8s.io/client-go v0.21.0-rc.0
-	k8s.io/code-generator v0.21.0-rc.0
-	k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
+	k8s.io/code-generator v0.22.1
```

Comment:
I think these changes are still wrong and need to be reverted

Reply:
Changed back to v0.21.0-rc.0